static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -248,9 +248,15 @@ const STATES = {
 };
 
 const shouldAutoplay = (atomId: string): boolean => {
-    const isAutoplayBlockingPlatform = () => isIOS() || isAndroid();
+    const isUSContent =
+        config.get('page.productionOffice').toLowerCase() === 'us';
+    const isAutoplayBlockingPlatform = () =>
+        (isIOS() || isAndroid()) && !isUSContent;
 
     const isInternalReferrer = (): boolean => {
+        if (isUSContent) {
+            return true;
```

Comment:
The name of our boolean is `isInternalReferrer`. I don't think we should change the logic here, since then we make the function lie for us 🙊

Reply:
What we are probably hoping to achieve is not caring about the page being from an internal referrer for US content, so we probably want this check somewhere else (outside of this function) 😄

Reply:
Yap you're right. I have now refactored all the conditions of US to its own block which doesn't include `isInternalReferrer`  the referred check or the `isAutoplayBlockingPlatform`